woocommerce/woocommerce-gutenberg-products-block
# Add empty content messages

Forces the product grid block renderer to return nothing if no products are found so that a message is displayed.


Fixes #686

### Screenshots

![Edit Post ‹ local wordpress test — WordPress 2019-07-10 16-29-16](https://user-images.githubusercontent.com/90977/60983011-228e8000-a331-11e9-80db-3040ee5638be.png)

After/if https://github.com/WordPress/gutenberg/pull/16512 is merged, we get this instead:

![Edit Post ‹ local wordpress test — WordPress 2019-07-10 16-27-25](https://user-images.githubusercontent.com/90977/60983041-2de1ab80-a331-11e9-84b0-0de70cfca58e.png)

Blocked by that PR.

### How to test the changes in this Pull Request:

1. Select a category with no products and save
2. See result

### Changelog

> Show a message in the editor if no products are found rather than show nothing.
